znai-docs/znai/flow/support.md

```diff
@@ -0,0 +1,5 @@
+# Definition
+
+It is very useful to provide a way to contact an owner of a documentation, so he/she can get a feedback about the documentation from a reader. When a user defines a link in meta.json, Support link will show up under a page title if you define a `link` under `support`. 
+
+:include-json: meta.json {highlight: [9, 10, 11]}
```

Comment:
highlight by text is fixed

Reply:
Yeah but I want to highlight the area, not just one variable.